templates/error.html

```diff
@@ -71,9 +71,12 @@
     </div>
   </nav>
   <div class="not-found-page">
-    <p class="content">Sorry, this page couldn't be found.</p>
-    <a class="content" href="/">Return home</a>
-    <a class="content" href="/app/search">Perform a search</a>
+    <img src="static/img/logo-green.png" alt="logo" class="logo-image" />
```

Comment:
The way you've linked the static image does not work for anything aside from base paths. For example, running locally visit http://localhost:5000/contributors/9. The image tries to load http://localhost:5000/contributors/static/img/logo-green.png which doesn't exist.

<img width="1280" alt="screen shot 2018-07-25 at 10 21 16 pm" src="https://user-images.githubusercontent.com/9062287/43243029-27377a26-9059-11e8-9cbd-84c3c709f1ed.png">

Reply:
Good catch.